### PR TITLE
Adding publish-over-ssh to COMPATIBILITY.md.

### PR DESCRIPTION
**Pull request by abayer**
Also fixed the performance plugin checkbox.

### CODE REVIEW

**Comment by abayer** [comment]
cc @reviewbybees

**Comment by oleg-nenashev** [comment]
:bee:

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -57,8 +57,9 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `GitHubCommitNotifier`, `GitHubSetCommitStatusBuilder` (`github`): scheduled to be supported as of 1.14.3
 - [ ] `CoverityPublisher` (`coverity`): [JENKINS-32354](https://issues.jenkins-ci.org/browse/JENKINS-32354)
 - [X] `XUnitPublisher` and `XUnitBuilder` (`xunit`): scheduled to be supported as of 1.100
-- [] `PerformancePublisher` (`performance`) [JENKINS-32650](https://issues.jenkins-ci.org/browse/JENKINS-32650)
+- [ ] `PerformancePublisher` (`performance`) [JENKINS-32650](https://issues.jenkins-ci.org/browse/JENKINS-32650)
```
Comment (jglick) [inline COMPATIBILITY.md]: missing `:`

File: COMPATIBILITY.md [hunk 2]
```diff
@@ -57,8 +57,9 @@ Entries list the class name serving as the entry point to the relevant functiona
...
 - [ ] `zephyr-for-jira-test-management` : [JENKINS-32801](https://issues.jenkins-ci.org/browse/JENKINS-32801)
+- [ ] `PublishOverSSH`: [JENKINS-27963](https://issues.jenkins-ci.org/browse/JENKINS-27963)
```
Comment (jglick) [inline COMPATIBILITY.md]: missing plugin name

```markdown
(`publish-over-ssh`)
```

Comment (jglick) [inline COMPATIBILITY.md]: I can find no such class in GitHub. I see `BapSshPublisher`. Is that what you meant?

Comment (abayer) [inline COMPATIBILITY.md]: I was just spitballing. Fixing.

**Pull request merged**